app/bundles/FormBundle/Entity/Form.php

```diff
@@ -780,8 +780,8 @@ public function setFormType($formType)
      */
     public function setNoIndex($noIndex)
     {
-        $this->isChanged('noIndex', $noIndex);
-        $this->noIndex = $noIndex;
+        $this->isChanged('noIndex', (bool) $noIndex);
```

Comment:
This may cause problem as the column is nullable. So it should allow all 3 values: true, false, null. This change will retype null to false. How about something like 
```suggestion
        $this->isChanged('noIndex', null === $noIndex ? null : (bool) $noIndex);
```
Applies for the other bool retypes in this PR.

Reply:
I'm looking into this. Fix coming soon.